### PR TITLE
CFM-1289: Implement leave functionality for groups via CTA

### PR DESCRIPTION
**Pull request by DavidHernandez**
Previous pull request: https://github.com/capacity4dev/capacity4more/pull/1060
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-1289

Picture:
ctas

## Depends on https://github.com/capacity4dev/capacity4more/pull/1120

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module [hunk 1]
```diff
@@ -645,6 +653,35 @@ function c4m_user_og_group_actions_cta($group) {
...
+ */
+function c4m_user_og_group_management_cta($group) {
+  $items = array();
+
+  if (_c4m_features_og_members_is_power_user($group)) {
+    $items['#title'] = t("You're the manager of this group");
+    $items['#links'][] = array(
+      '#theme' => 'c4m_group_text_cta',
+      '#icon' => 'fa-cogs',
+      '#text' => l(t('Manage membership'), 'manage'),
```
Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: Manage group or edit membership?

**Comment by DavidHernandez** [comment]
Screenshots:
### Desktop
cta-desktop
### Mobile
cta-mobile
### Tablet
cta-tablet

**Comment by ordavidil** [comment]
@DavidHernandez 
please re-roll

**Comment by ordavidil** [comment]
@DavidHernandez 
please re-roll and make sure the functionality still works

**Comment by ordavidil** [comment]
@DavidHernandez 
please re-roll

**Comment by ordavidil** [comment]
@DavidHernandez 
make sure the owner of a group also has the "edit membership" link.

**Pull request merged**